integration/test_number_of_indexes_on_replica_fullsync.py

```
@@ -0,0 +1,39 @@
+    )
+    def test_number_of_indexes_after_full_sync(self):
+        primary_client = self.get_primary_connection()
+        replica_client = self.get_replica_connection(0)
+
+        idx.create(primary_client, True)
+        idx.load_data(primary_client, NUM_DOCS)
+        assert primary_client.info("search")["search_number_of_indexes"] == 1
+
+        replica_num_indexes = lambda: replica_client.info("search").get(
+            "search_number_of_indexes"
+        )
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `replica_num_indexes` as a `def`

(E731)